spec.md

```diff
@@ -2041,6 +2041,8 @@ s       any             as if by str(x)
 r       any             as if by repr(x)
 d       number          signed integer decimal
 o       number          signed octal
```

Comment:
Optional: Clarify that no preceding `0x` or `0o` notation is emitted?

Reply:
Good point; done.

The usual way to get the prefix is `'%#x'` but Starlark formatting doesn't yet support flags. (`'0x%x'` is wrong for negative numbers.)